packages/paste-core/components/combobox/src/styles/ComboboxEmptyState.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import type {BoxProps} from '@twilio-paste/box';
+import {ComboboxListboxOption} from './ComboboxListboxOption';
+import {ComboboxListboxGroup} from './ComboboxListboxGroup';
+
+export interface ComboboxEmptyStateProps {
+  element?: BoxProps['element'];
+}
+
+const ComboboxEmptyState = React.memo(
+  React.forwardRef<HTMLUListElement, ComboboxEmptyStateProps>(({element = 'COMBOBOX', ...props}, ref) => {
+    return (
+      <ComboboxListboxGroup element={element} ref={ref}>
+        <ComboboxListboxOption {...props} element={`${element}_EMPTY_STATE`} variant="default" disabled>
```

Comment:
I think the resolution here was that we're going for composition, over doing empty state for the consumer. Because in some cases it can be disabled, in others it can't.

These will be covered in examples on the doc page

Reply:
Deleted this component. It is now a prop passed to the component as `EmptyState`